src/util/createMetaModule.js

```diff
@@ -0,0 +1,60 @@
+module.exports = function createMetaModule(mapFunction, moduleOptions){
+
+  moduleOptions = moduleOptions || {};
+  moduleOptions.infoJson = moduleOptions.infoJson || 'info.json';
+
+  function MetaModule(options, UI) {
+
+    var defaults = require('./getDefaults.js')(require(infoJson));
```

Comment:
I think this should be moduleOptions.infoJson instead of directly mentioning infoJson